src/App.test.js

```diff
@@ -10,7 +10,7 @@ import logger from "redux-logger";
 import { Provider } from "react-redux";
 import { reducer as plaidReducer } from "./redux/reducers/PlaidReducer";
 import { reducer as registerReducer } from "./redux/reducers/RegisterReducer";
-import { reducer as profileReducer } from "./redux/reducers/ProfileReducer";
+// import { reducer as profileReducer } from "./redux/reducers/ProfileReducer";
```

Comment:
If we don't need this, can we delete this comment? Looks like ProfileReducer file doesn't exist at all